provider/machinepool/machine_pool_resource.go

```diff
@@ -184,6 +189,11 @@ func (r *MachinePoolResource) Schema(ctx context.Context, req resource.SchemaReq
 					stringplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"aws_subnet_ids": schema.ListAttribute{
```

Comment:
Consider to remove the "aws_" prefix.
I understand that it better to have this, but maybe in this case it preferred to have it similar to "sunbnet_id" attribute

Reply:
I tried to be aligned with cluster rosa resource

Reply:
I changed it to subnet_ids